libs/remix-lib/src/execution/execution-context.ts

```diff
@@ -96,15 +117,15 @@ export class ExecutionContext {
     this.executionContext = null
     this.blockGasLimitDefault = 4300000
     this.blockGasLimit = this.blockGasLimitDefault
-    this.currentFork = 'muirGlacier'
+    this.currentFork = 'berlin'
     this.vms = {
       /*
```

Comment:
for the moment we are always using the last fork.

Reply:
We would also be a big fan of adding the option to choose from minimally the last 2 forks or so on the `EthereumJS VM` in the UI. This would greatly beneficial for the community especially during the times of transition. With all this base work you have done here now maybe we can tackle the integration for London even faster and - in the best case - have a London - and especially EIP-1559-ready - Remix out *before* the HF occurs so that people could test their stuff on testnets with the new gas logic? That would be pretty cool! 😄 

Not sure if someone has the capacity to do, but would you generally be open for such a PR from someone from our team adding this additional switch logic?

So in the UI that would at the end look something like this:

![image](https://user-images.githubusercontent.com/931137/116540503-86a97c00-a8ea-11eb-9877-8ad99957b1cb.png)

Reply:
indeed ;) we'll check what can be done